### ISSUE TITLE
error: module 'keras.engine' has no attribute 'Layer'

### ISSUE THREAD

**Issue by HaixinLiuNeuro** [navidyou/Mask-RCNN-implementation-for-cell-nucleus-detection-executable-on-google-colab-#2]
![image](https://user-images.githubusercontent.com/46263698/227015011-aa185867-7dd0-43c4-af0e-70a6908a1960.png)

it seems to be a issue associated with Colab.

**Comment by ViirajPatel** [comment]
Replace KE. to KL. in model.py.
